dash/dash.py

```diff
@@ -300,6 +300,8 @@ def index(self, *args, **kwargs):  # pylint: disable=unused-argument
         <html>
             <head>
                 <meta charset="UTF-8">
+                <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
My dash app doesn't work with IE. Only after I added these two lines, it work with all browsers. It will be nice to include this in the Dash by default.